.template/addons/github/.github/workflows/review_code_using_danger.yml.tt

```diff
@@ -0,0 +1,57 @@
+name: Automated code review
+
+on:
+  pull_request:
+
+env:
+  DANGER_GITHUB_API_TOKEN: ${{ github.token }}
+
+jobs:
+  automated_code_review:
+    name: Run Danger
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      <%- if WEB_VARIANT -%>
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '<%= NODE_VERSION %>'
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Yarn cache
+        uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Node modules cache
+        id: node-modules-cache
+        uses: actions/cache@v2
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-nodemodules-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-nodemodules-
+
+      - name: Yarn install
+        if: steps.yarn-cache.outputs.cache-hit != 'true' || steps.node-modules-cache.outputs.cache-hit != 'true'
+        run: yarn
+
+      <%- end -%>
+      - name: Run Danger
+        run: bundle exec danger
```

Comment:
should we have the bundle install step and cache the bundle path as well 🤔

Reply:
it isn't necessary, the step to [setup Ruby](https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby#single-job) handled that already (line 21)
https://github.com/nimblehq/rails-templates/blob/df4f3b35bf54a3a8747c85cf5d97361b966c80e9/.template/addons/github/.github/workflows/review_code_using_danger.yml.tt#L18-L21
according to the document of that Github action
```
bundler-cache: true # runs 'bundle install' and caches installed gems automatically
```

Reply:
Oh I see, good to know that @longnd